be/src/main/java/yeonba/be/arrow/service/ArrowService.java

```diff
@@ -40,16 +45,59 @@ public void dailyCheck(long userId) {
         dailyCheckUser,
         DAILY_CHECK_ARROW_COUNT
     );
-    arrowTransactionCommandRepository.save(arrowTransaction);
+    arrowCommand.save(arrowTransaction);
 
     dailyCheckUser.updateLastAccessedAt(dailyCheckedAt);
     dailyCheckUser.addArrow(DAILY_CHECK_ARROW_COUNT);
   }
 
+  /*
+    화살 보내기 비즈니스 로직은 다음 과정을 거친다.
+    1. 자기 자신에게 화살을 보내는 상황 검증
+    2. 화살을 보낸 사용자에게 또 보내는 상황 검증
+    3. 화살 내역 저장
+    4. 보내는 사용자 화살 감소, 화살이 부족할 경우 예외 발생
+    5. 받는 사용자 화살 증가
+
+   */
+  @Transactional
+  public void sendArrow(
+      long senderId,
+      long recipientId,
+      ArrowSendRequest request) {
+
+    if (senderId == recipientId) {
+      throw new GeneralException(ExceptionType.CAN_NOT_SEND_ARROW_SELF);
+    }
+
+    User sender = userService.findById(senderId);
+    User recipient = userService.findById(recipientId);
```

Comment:
사용자의 존재 여부 검증이 먼저 되고 동일 사용자 검증을 하는 것이 자연스러운 흐름인 것같은데 어떻게 생각하시나요?
```suggestion

    User sender = userService.findById(senderId);
    User recipient = userService.findById(recipientId);

    if (senderId == recipientId) {
      throw new GeneralException(ExceptionType.CAN_NOT_SEND_ARROW_SELF);
    }

```

Reply:
동의합니다. 수정하여 반영토록 하겠습니다.